src/Task2Test.kt

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+internal class Task2Test {
+
+    @Test
+    //Exceptional case where index -1 is returned
```

Comment:
Strange comment. You keep refering at this as indexes but they are actually values.